components/builder-web/app/actions/builds.ts

```diff
@@ -59,11 +59,13 @@ export function fetchBuildLog(id: string, token: string, start = 0) {
       .then((data) => {
           dispatch(populateBuildLog(data));
 
-          if (data["is_complete"] && start !== 0) {
+          let complete = data["is_complete"];
+
+          if (complete && data["start"] !== 0) {
```

Comment:
Almost -- `fetchBuild` fetches the _job_ in 5 seconds (not the log, just the job metadata) in order to reflect its final state in the UI.